tutorials/continuous-deployment-pipeline-google-container-engine-with-codeship/index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Continuous Deployment Pipeline to Google Container Engine using Codeship
+description: Learn how to create a continuous deployment pipeline to Container Engine from CodeShip.
+author: kellyjandrews
+tags: CD, Container Engine, CodeShip, Pipeline
+date_published: 2017-08-28
+---
+This is a walkthrough of steps necessary to create a Continuous Deployment
+pipeline to Google Container Engine using Codeship. In this tutorial you will
+learn how to deploy a containerized application when new code is merged into the
+master branch and all integration tests have passed.
+
+Continuous Deployment is the practice of automating the delivery of software to
+production after passing a series of tests. Although not for everyone, it should
+be the goal of any software development and devops team not restricted by
+regulatory constraints.
+
+## Before you begin
+
+Take the following steps to enable the Google Container Engine API:
+
+1.  Visit the [Container Engine](https://console.cloud.google.com/projectselector/kubernetes)
+    page in the Google Cloud Platform Console.
+1.  Create or select a project in the [Google Cloud Platform
+    Console](https://console.cloud.google.com/).
+1.  Wait for the API and related services to be enabled, which can take several
+    minutes.
+1.  [Enable billing][billing] for your project.
+
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+
+Make sure you have the following:
+
+1.  [Docker CE Version 17.03.01](https://www.docker.com/community-edition)
+1.  [Codeship Account](https://app.codeship.com/registrations/new)
+1.  [Codeship Jet CLI](https://documentation.codeship.com/pro/builds-and-configuration/cli/)
+1.  [`hello-express` source code](https://github.com/kellyjandrews/hello-express)
+1.  [GitHub](https://github.com/), [Bitbucket](https://bitbucket.org/), [GitLab](https://about.gitlab.com/) account
+
+## Setting up the continuous deployment pipeline
+
+### Step 1: Create service account
+
+The interactions with the Google Cloud API from Codeship require a service
+account with permissions to the Storage and Container services.
+
+Follow these steps to create a service account:
+
+1.  Visit the [Service accounts](https://console.cloud.google.com/projectselector/iam-admin/serviceaccounts)
+    page in the Google Cloud Platform Console.
+1.  Select your project.
+1.  Click `Create Service Account`
+1.  Enter a name
+1.  Click `Select a Role` and choose the following permissions:
+
+    * Project -> Service Account Actor
```

Comment:
You can use `&rarr;` (&rarr;) character for better arrows.

Reply:
Thanks - I'll get these updated.